api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -215,7 +215,7 @@ public Order getOrder(Integer orderId) throws DAOException {
 		}
         Criterion fulfillerStatusCriteria = null;
         if (searchCriteria.getIncludeNullFulfillerStatus() != null ) {
-            if (searchCriteria.getIncludeNullFulfillerStatus().booleanValue() == true ) {
+            if (searchCriteria.getIncludeNullFulfillerStatus().booleanValue() ) {
```

Comment:
You forgot to delete the space in front of the last closing brace (the area marked with a red dot).
![image](https://user-images.githubusercontent.com/9057882/91304719-3ce02300-e7aa-11ea-94bb-0f4ba7f5af6c.png)

Reply:
oh okay let me change close it and do a git commit --amend for the message and git push --force for that space